src/test/java/org/folio/rest/impl/AllowedServicePointPathTest.java

```diff
@@ -0,0 +1,218 @@
+  public static final String CONTENT_TYPE_HEADER_NAME = "content-type";
+  private static final String CONTENT_TYPE_APPLICATION_JSON_HEADER = "application/json";
+  private static final String CIRCULATION_STORAGE_HEADER_NAME = "x-okapi-bad-user-id";
+  private static final String ECS_TLR_HEADER_NAME = "x-okapi-bad-data";
+  private static final String EMPTY_HEADER = "";
+  private static final String CIRCULATION_STORAGE_ENABLED_HEADER = "circulation-storage-true";
+  private static final String CIRCULATION_STORAGE_DISABLED_HEADER = "circulation-storage-false";
+  private static final String ECS_TLR_ENABLED_HEADER = "ecs-tlr-true";
+  private static final String ECS_TLR_DISABLED_HEADER = "ecs-tlr-false";
+  private static final String ECS_TLR_MOD_ENABLED_JSON_FILE_PATH =
+    "/ecs_tlr_module_feature_enabled.json";
+  private static final String ECS_TLR_MOD_DISABLED_JSON_FILE_PATH =
+    "/ecs_tlr_module_feature_disabled.json";
+  private static final String ECS_TLR_SETTINGS_PATH = "/tlr/settings";
+  private static final String CIRCULATION_STORAGE_MOD_ENABLED_JSON_FILE_PATH =
+    "/circulation_storage_module_feature_enabled.json";
+  private static final String CIRCULATION_STORAGE_MOD_DISABLED_JSON_FILE_PATH =
+    "/circulation_storage_module_feature_disabled.json";
+  public static final String ALLOWED_SERVICE_POINTS_CIRCULATION_JSON_FILE_PATH =
+    "/allowed_service_points_circulation.json";
+  public static final String ALLOWED_SERVICE_POINTS_CIRCULATION_BFF_JSON_FILE_PATH =
+    "/allowed_service_points_circulation_bff.json";
+  private static final String CIRCULATION_SETTINGS_STORAGE_PATH =
+    "/circulation-settings-storage/circulation-settings";
+  private static final String SERVICE_POINTS_CIRCULATION_BFF_RESPONSE_JSON =
+    "/allowed_service_points_circulation_bff_response.json";
+  private static final String CIRCULATION_REQUESTS_ALLOWED_SERVICE_POINTS_PATH =
+    "/circulation/requests/allowed-service-points";
+  private static final String CIRCULATION_BFF_ALLOWED_SERVICE_POINTS_PATH =
+    "/circulation-bff/allowed-service-points";
+  private static final String SERVICE_POINTS_CIRCULATION_RESPONSE_JSON =
+    "/allowed_service_points_circulation_response.json";
+  private static final String ACCOUNT_ID_PATH_PARAM_KEY = "accountId";
+  private static final String INSTANCE_ID_PATH_PARAM_KEY = "instanceId";
+
+  private static final String INTERNAL_SERVER_ERROR_STATUS_HEADER_VALUE = "status 500";
+  private static final String INTERNAL_SERVER_ERROR_STATUS_LINE = "HTTP/1.1 500 Internal Server Error";
+  private static final String RESPONSE_WITH_ERROR_EXPECTED = "java.lang.RuntimeException: " +
+    "java.util.concurrent.CompletionException: io.vertx.core.impl.NoStackTraceTimeoutException: " +
+    "The timeout period of 5000ms has been exceeded while executing GET /tlr/settings " +
+    "for server null";
+
+  @BeforeEach
+  @Override
+  public void setUp(Vertx vertx, VertxTestContext context) {
+    super.setUp(vertx, context);
```

Comment:
What is the point of overriding a method if you are explicitly calling it anyway? You can mark both methods with `@BeforeEach`, and the method in super-class will be called before method of a child class. Just make sure that methods have different names.

Reply:
refactored